frontend/src/organizations/components/OrganizationMembersTable.js

```diff
@@ -29,7 +29,7 @@ const OrganizationMembersTable = (props) => {
     minWidth: 200
   }, {
     accessor: item => item.email,
-    className: 'col-status-display',
+    className: 'col-email',
```

Comment:
This was just something I noticed.  The column was used for email, but had an odd 'status-display' class.

Reply:
good catch